### PR TITLE
move shared inc files to recipes-domx

### PR DESCRIPTION
**Pull request by varder**
The files xen-version.inc and xt_shared_env.inc are copying into
meta-xt-images repository during domain configuration.
To avoid unnecessary files copying, move include files to
the separate layer.

PR should be merged with xen-troops/meta-xt-images#189

### CODE REVIEW

**Comment by rshym** [comment]
Prod-aos built successfully with https://github.com/xen-troops/meta-xt-images/pull/189

**Comment by rshym** [comment]
Build of `xt-update` failed. Seems like issue is not in PR, but some info need to be provided in product config.
Need to clarify with AOS team before merge of PR.
Error message (without traces)
```
Exception: RuntimeError: mandatory field (id or fileName or vendorVersion) is missing
```

File: recipes-xt-image/xt-image/xt-update.bb [hunk 1]
```diff
@@ -5,10 +5,6 @@ LICENSE = "Apache-2.0"
 
 inherit metadata-generator bundle-generator rootfs-image-generator
 
-# Require
```
Comment (al1img) [inline recipes-xt-image/xt-image/xt-update.bb:8]: Why this include is removed, it is required to create this recipe.

Comment (varder) [reply recipes-xt-image/xt-image/xt-update.bb:8]: fixed

**Comment by rshym** [comment]
Tested build of this PR #111 with https://github.com/xen-troops/meta-xt-images/pull/189
`prod-aos:xt-image` and `prod-aos:xt-update` built successfully.

**Comment by rshym** [comment]
@al1img Could you please look into PR one more time? I have tested compilation - xt-image and xt-update built OK.
If no objections - let's merge aos, devel and images.

**Comment by rshym** [comment]
`Reviewed-by: Ruslan Shymkevych <ruslan_shymkevych@epam.com>`

**Pull request merged**